app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,18 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      if user_params[:to_confirm] == 'true'
+        params[:user].delete(:to_confirm)
+        params[:user][:confirmed_at] = Time.now
+      elsif user_params[:to_confirm] == 'false'
+        params[:user].delete(:to_confirm)
+        params[:user][:confirmed_at] = nil
+      end
+      @user.update_attributes(user_params)
```

Comment:
@ChrisBr The botstrap-switch initiates a request from javascript. `redirect_to` doesn't work in this situation. What should be the best way to handle the this?